spec/imsc-hrm.html

```diff
@@ -985,7 +983,19 @@ <h2>Acknowledgements</h2>
   <section id="substantive-changes-summary" class="appendix informative">
     <h2>Summary of substantive changes</h2>
 
-   
+    <section>
+      <h3>Reduce empty ISD complexity to zero</h3>
+      <p>In order to allow short (less than 100 ms) gaps between subtitles, which is common practice, the complexity of
+      presenting <a>empty ISDs</a> has been reduced to zero: instead of being drawn into the Back Buffer, an <a>empty
+      ISD</a> merely disconnects the Front Buffer from the display while it is presented.</p>
+    </section>
+
+    <section>
+      <h3>Complexity of clearing the Back Buffer applies to all non-empty ISDs</h3>
+      <p>The first <a>Intermediate Synchronic Document</a> is no longer treated differently and incurs a cost for
+      clearing the Back Buffer.</p>
+    </section>
```

Comment:
Is it worth also adding that a known consequence of this is that any non-empty ISD beginning at time 0 will always fail?

Reply:
If a non-empty ISD is the first, then it has at most IPD seconds to draw.

Reply:
Sorry, yes.

Reply:
Happy with the current text then?

Reply:
Yes I think so. Mental note to self: when/if we extend the scope of this to apply to ISDs arising from a sequence of documents, this means that the ISDs for each document need to be ready for their rendering to be scheduled no later than IPD before the effective begin time of the document. At the moment this is something we don't consider: we assume that the document is available and parsed into ISDs even before the user begins playback of the related video object, but that position is harder (but not impossible) to maintain with a sequence, for example as might be provided in fragmented MP4. We could of course delegate that to upstream processing, but it does seem like something to note down somewhere...